src/components/application_manager/src/commands/mobile/create_interaction_choice_set_request.cc

```diff
@@ -267,12 +344,33 @@ void CreateInteractionChoiceSetRequest::SendVRAddCommandRequests(
       }
     }
 
-    msg_params[strings::cmd_id] =
-        choice_set[strings::choice_set][chs_num][strings::choice_id];
-    msg_params[strings::vr_commands] =
-        smart_objects::SmartObject(smart_objects::SmartType_Array);
-    msg_params[strings::vr_commands] =
-        choice_set[strings::choice_set][chs_num][strings::vr_commands];
+    const SmartObject& choice = choice_set[strings::choice_set][chs_num];
+
+    if (!choice[strings::vr_commands].asArray() ||
+        choice[strings::vr_commands].empty()) {
+      --expected_chs_count_;
```

Comment:
@LevchenkoS I see you not the one named this variable like this, but if is not a problem for you could you rename it to be more meaningful ?